justfile

```diff
@@ -46,6 +46,7 @@ default:
 
 # Install necessary tools
 install-tools:
```

Comment:
Now that we're publishing prebuilt images, I'm even wondering if we should just invoke these `just` setup tasks as part of the image build itself. Even the `setup-integration-test` task as well.

Reply:
We can add the Rust nightly toolchain to the devcontainer build step, but not the integration test setup as that will install the python virtual environment (which is placed inside the workspace and not in the image itself)

Reply:
Done in 77b214ff. I haven't removed this task as it'd still be useful for those not using a devcontainer.